src/Internal.AspNetCore.Sdk/build/Workarounds.targets

```diff
@@ -0,0 +1,49 @@
+<!--
+WARNING: These targets are indented for building Microsoft's ASP.NET Core repos, and is not intended
+for use outside of Microsoft.
+-->
+
+<!-- this file is dedicated to working around bugs in Microsoft.NET.Sdk RC.2 release -->
+<Project>
+    <!-- workaround https://github.com/Microsoft/vstest/issues/191 -->
+    <PropertyGroup
+        Condition="$(MSBuildProjectName.EndsWith('Test')) OR $(MSBuildProjectName.EndsWith('Tests'))">
+        <OutputType Condition="$(TargetFramework.StartsWith('netcoreapp'))">exe</OutputType>
+    </PropertyGroup>
+    <!-- end workaround -->
+
+    <!-- workaround https://github.com/NuGet/Home/issues/4063 -->
+    <!-- workaround https://github.com/NuGet/Home/issues/3953 -->
+    <Target Name="ResolveActualPackageVersions"
+            BeforeTargets="GenerateNuSpec"
+            DependsOnTargets="ResolvePackageDependenciesDesignTime">
+
+        <ItemGroup>
+            <_ProjectReferences Update="@(_ProjectReferences)" Condition="'%(PackageVersion)'==''">
+                <PackageVersion>$(Version)</PackageVersion>
+            </_ProjectReferences>
+
+            <_FloatingVersions Include="@(_PackageReferences)" Condition="$([System.String]::new('%(Version)').EndsWith('*'))" />
+            <_ResolvedFloatingVersion Include="@(_FloatingVersions)">
+                <ResolvedName>%(_DependenciesDesignTime.Name)</ResolvedName>
+                <Version>%(_DependenciesDesignTime.Version)</Version>
+            </_ResolvedFloatingVersion>
+            <_PackageReferences Remove="@(_PackageReferences)" Condition="$([System.String]::new('%(Version)').EndsWith('*'))" />
+            <_PackageReferences Include="@(_ResolvedFloatingVersion)" Condition="'%(Identity)'=='%(_ResolvedFloatingVersion.ResolvedName)'" />
+        </ItemGroup>
+
+        <Message Importance="Normal" Text="Proj: %(_ProjectReferences.FileName) %(_ProjectReferences.PackageVersion)" />
```

Comment:
Sure, but it doesn't hurt to leave it. Doesn't show up by default in log output.